schutzbot/ci_details.sh

```diff
@@ -16,21 +19,36 @@ CI MACHINE SPECS
 ------------------------------------------------------------------------------
 
      Hostname: ${HOSTNAME}
+         User: ${USER}
    Primary IP: ${PRIMARY_IP}
   External IP: ${EXTERNAL_IP}
   Reverse DNS: ${PTR}
          CPUs: ${CPUS}
           RAM: ${MEM} GB
          DISK: ${DISK} GB
+         ARCH: ${ARCH}
+       KERNEL: ${KERNEL}
 
 ------------------------------------------------------------------------------
 EOF
 echo -e "\033[0m"
 
+echo "List of system repositories:"
+yum repolist -v
+
+echo "------------------------------------------------------------------------------"
+
 echo "List of installed packages:"
 rpm -qa | sort
 echo "------------------------------------------------------------------------------"
 
+if rpm --quiet -q python36; then
+    echo -e "\n FAIL: python36 is installed, see #794 ..."
```

Comment:
This check is no longer relevant, `osbuild` on EL8 actually requires `python36` now so the check is actually apparently broken:


```
# dnf -q repoquery --requires osbuild-53-2.el8
(osbuild-selinux if selinux-policy-targeted)
/bin/sh
/usr/bin/python3
/usr/libexec/platform-python
bash
bubblewrap
coreutils
curl
dnf
e2fsprogs
glibc
policycoreutils
python3-osbuild = 53-2.el8
qemu-img
systemd
tar
util-linux
```

(`/usr/bin/python3` is provided by `python36` on el8)

Reply:
Ok I just removed it. Idk what's the original reason the check was there but it's in `osbuild-composer` repository and we decided with Alex to just keep it since it was there for a reason apparently.